packages/lessons-worker/src/lib/logging/file-logger.ts

```
@@ -34,19 +34,27 @@ winston.addColors(colors);
+// Format for console output with emojis and better readability
 const consoleFormat = winston.format.combine(
 	winston.format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss:ms' }),
 	winston.format.colorize({ all: true }),
-	winston.format.printf(
-		(info) =>
-			`${info.timestamp} ${info.level}: ${info.message} ${
-				info.metadata ? JSON.stringify(info.metadata) : ''
-			}`
-	)
+	winston.format.printf((info) => {
+		const emoji = levelEmojis[info.level.replace(/\u001b\[[0-9;]*m/g, '') as keyof typeof levelEmojis] || '';
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)